packages/grafana-ui/src/components/DateTimePickers/TimeRangePicker.tsx

```diff
@@ -153,8 +155,14 @@ export function TimeRangePicker(props: TimeRangePickerProps) {
         aria-label={t('time-picker.range-picker.backwards-time-aria-label', 'Move time range backwards')}
         variant={variant}
         onClick={onMoveBackward}
-        icon="angle-left"
+        icon="angle-double-left"
         type="button"
+        iconSize="xl"
+        tooltip={
```

Comment:
if you're going to specify `tooltip`, you don't need to specify `aria-label` as well